From 73bc3ea559c7324f47d4283d56a42d1840ec4832 Mon Sep 17 00:00:00 2001
From: Dave Eddy <dave@daveeddy.com>
Date: Tue, 12 Mar 2019 12:22:20 -0400
Subject: [PATCH] joyent/boray#6 want `make check` Reviewed by: Kelly
 McLaughlin <kelly.mclaughlin@joyent.com> Approved by: Kelly McLaughlin
 <kelly.mclaughlin@joyent.com>

---
 Makefile | 7 +++++++
 1 file changed, 7 insertions(+)
 create mode 100644 Makefile

diff --git a/Makefile b/Makefile
new file mode 100644
index 0000000..06ff1d7
--- /dev/null
+++ b/Makefile
@@ -0,0 +1,7 @@
+#
+# Copyright 2019 Joyent, Inc
+#
+
+.PHONY: check
+check:
+	cargo check
-- 
2.21.0

